src/locales/polyglot.js

```diff
@@ -1,23 +1,29 @@
 import Polyglot from 'node-polyglot'
-import en from './en.json'
-import es from './es.json'
-import enMobile from './mobilePhrases/en.json'
-import esMobile from './mobilePhrases/es.json'
+import en_US from './en_US/en_US.json'
+import es_ES from './es_ES/es_ES.json'
+import de_DE from './de_DE/de_DE.json'
 import { isDesktop } from '~utils'
 import { memoize } from '~utils/func'
 
-const defaultLocaleTag = 'en'
+const defaultLocaleTag = 'en_US'
 
 // Language tags should follow the IETF's BCP 47 guidelines, link below:
 //https://www.w3.org/International/questions/qa-lang-2or3
 // Generally it should be a two or three charaters tag (language) followed by a two/three characters subtag (region), if needed.
-const availableTransations = {en, es}
+const availableTransations = {en_US, es_ES, de_DE}
 
-const mobileTranslations = {
-  en: enMobile,
-  es: esMobile
+const mobilePhrases = () => {
```

Comment:
Would be better if the function name describes what it does instead of what it returns, e.g. `getMobilePhrases`